ambry-replication/src/test/java/com.github.ambry.replication/ReplicationTest.java

```diff
@@ -70,6 +70,7 @@
 import java.nio.channels.WritableByteChannel;
 import java.util.ArrayList;
 import java.util.Arrays;
```

Comment:
Note: There are changes to the tests, but the diff is large because I removed unnecessary try-catch blocks which changed the indentation. The indentation change has been isolated into the second commit (which is just about removing the try-catch). To review the actual change to the tests, just look at the first commit.